src/platforms/esp32/components/avm_builtins/network_driver.c

```diff
@@ -404,10 +410,15 @@ static wifi_config_t *get_sta_wifi_config(term sta_config, GlobalContext *global
     return wifi_config;
 }
 
-static char *get_default_device_name()
+static char *get_default_device_name(esp_mac_type_t interface)
 {
     uint8_t mac[6];
+    #if ESP_IDF_VERSION_MAJOR == 5
```

Comment:
I seem to remember that our clang format rules were enforcing left justification of these #ifdefs.  Regardless, we might want to establish a convention.

Reply:
`clang-format-13 -i` does want to make those left justified, and it found a few other problems in this file. Apparently I completely forgot to check the formatting before submitting this PR.